spring-boot/proxy/src/main/java/eu/coatrack/proxy/security/ApiKeyAuthTokenVerifier.java

```diff
@@ -150,7 +150,7 @@ private boolean isApiKeyValid(String apiKeyValue) throws AuthenticationException
                 log.debug("API key value is invalid: " + apiKeyValue);
                 return false;
             } else {
-                log.error("Error when communicating with auth server", e);
+                log.error("An unexpected error occurred when your CoatRack gateway contacted the central CoatRack portal in order to verify an API key that was sent by a client: Service is not unavailable/accessible, please check if the service is accessible and gateway config in https://coatrack.eu/admin/proxies", e);
```

Comment:
The message that the service is not accessible needs to be moved to the appropriate place, this error message is one step earlier, when checking the API key towards Coatrack Admin

Reply:
The code was now changed to the right place and it is working correctly.
Below are some screenshots of the testing:

![image](https://user-images.githubusercontent.com/47450617/92491595-b2101700-f1f2-11ea-9c45-1742d5833982.png)
![image](https://user-images.githubusercontent.com/47450617/92491628-bb997f00-f1f2-11ea-84ab-ada8586fe14f.png)